engines/sludge/sound.cpp

```diff
@@ -315,7 +315,7 @@ int SoundManager::makeSoundAudioStream(int f, Audio::AudioStream *&audiostream,
 
 	Common::SeekableReadStream *readStream = g_sludge->_resMan->getData();
 	uint curr_ptr = readStream->pos();
-	Audio::RewindableAudioStream *stream = Audio::makeWAVStream(readStream->readStream(length), DisposeAfterUse::NO);
+	Audio::SeekableAudioStream *stream = Audio::makeVorbisStream(readStream->readStream(length), DisposeAfterUse::NO);
```

Comment:
As discussed, this is not a proper fix. It breaks the WAVE playback.

Please fix it as proposed, by peeking into the file header and force-push (no need to close this PR, GitHub will pick up your push automatically)